build.gradle.kts

```diff
@@ -24,6 +24,13 @@ plugins {
     // since build-plugins also has <some> version in its dependency closure
     alias(libs.plugins.kotlin.multiplatform) apply false
     alias(libs.plugins.kotlin.jvm) apply false
+    alias(libs.plugins.aws.kotlin.repo.tools.artifactsizemetrics)
+}
+
+artifactSizeMetrics {
+    artifactPrefixes = setOf(":codegen", ":runtime")
```

Comment:
question: Do we care about the codegen artifact sizes? This is not something that customers are affected by

Reply:
I think we can get rid of them 👍